types/milesight/VS132/default.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$id": "https://akenza.io/xovis/VS132/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "totalCounterIn": {
```

Comment:
what's the difference between total and periodic?

Reply:
Total is adding counts since sensor start. While periodic counts are only counting the ones in the last timeframe.